tony-core/src/main/java/com/linkedin/tony/TonyClient.java

```diff
@@ -280,8 +276,12 @@ public boolean init(String[] args) throws ParseException, IOException {
     amMemory = Integer.parseInt(Utils.parseMemoryString(amMemoryString));
     amVCores = tonyConf.getInt(TonyConfigurationKeys.AM_VCORES,
         TonyConfigurationKeys.DEFAULT_AM_VCORES);
-    amGpus = tonyConf.getInt(TonyConfigurationKeys.AM_GPUS,
+    int amGpus = tonyConf.getInt(TonyConfigurationKeys.AM_GPUS,
         TonyConfigurationKeys.DEFAULT_AM_GPUS);
+    if (amGpus > 0) {
+      LOG.error("AM_GPUS not allowed on this version AM_GPUS [" + amGpus + "]");
```

Comment:
Maybe we should not fail if user requests AM GPUs? We can just change the error to a warning, set amGpus = 0, and continue?